### PR TITLE
Updated language files

### PR DESCRIPTION
**Pull request by Wertzui123**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
This pr removed a duplicated line in the english messages file and translated the usages to german in the german messages file

## Optional information
{Type something here}

### CODE REVIEW

File: resources/eng.ini [hunk 1]
```diff
@@ -195,7 +194,7 @@ warp.success=You are teleported to plot {%0}
 
 ; The /p middle Subcommand
 middle.name=middle
-middle.alias=mid
+middle.alias=middle
```
Comment (jasonwynn10) [inline resources/eng.ini]: This needs reverted. mid is the alias.

File: resources/deu.ini [hunk 2]
```diff
@@ -186,31 +186,31 @@ give.received={%0} gab dir das Grundstück: {%1}
...
+warp.usage=/p warp <id: int> [welt: string]
 warp.wrongid="Die Grundstück ID sollte in diesem Format sein: X;Z"
 warp.notinplotworld=Du bist nicht in einer Grundstückswelt!
 warp.unclaimed=Du kannst dich nicht zu einem Grundstück teleportieren, das niemanden gehört.
 warp.success=Du wurdest zu dem Grundstück {%0} teleportiert!
 
 ; The /p middle Subcommand
 middle.name=middle
-middle.alias=mid
+middle.alias=middle
```
Comment (jasonwynn10) [inline resources/deu.ini]: Needs reverted. Mid is the alias

Comment (Wertzui123) [reply resources/deu.ini]: Oh lol, I thougt I edit something else 🤦‍♂

File: resources/eng.ini (hunk 1 above)
Comment (Wertzui123) [reply resources/eng.ini]: Oh lol, I thougt I edit something else 🤦‍♂

**Comment by HimbeersaftLP** [comment]
Did you not capitalise the nouns in the arguments on purpose? (Spieler, Welt, etc.)

**Comment by Wertzui123** [comment]
Yes, I like spieler more than Spieler (even I know it should be with an uppercase letter)

**Pull request merged**